tokio-rustls/src/server.rs

```diff
@@ -123,3 +128,23 @@ where
         stream.as_mut_pin().poll_shutdown(cx)
     }
 }
+
+#[cfg(unix)]
```

Comment:
Should this also be implement for `tokio_rustls::TlsStream` and `tokio_rustls::client::TlsStream`?